lib/resources/example_json/r4_examples_procedure.rb

```diff
@@ -102,6 +102,278 @@ module Resources
       ]
     }.freeze
 
+    R4_PROCEDURE_PATIENT_ENTRY ||= {
+      "resourceType": 'Procedure',
+      "id": '2572581295',
+      "meta": {
+        "versionId": '0',
+        "lastUpdated": '2020-09-25T20:13:56Z'
+      },
+      "text": {
+        "status": 'generated',
+        "div": '<div xmlns="http://www.w3.org/1999/xhtml"><p><b>Procedure</b></p><p><b>Patient</b>: SMART, NANCY</p><p><b>Procedure</b>: Adenoid excision (text)</p><p><b>Date</b>: Apr  5, 2019</p><p><b>Status</b>: Completed</p><p><b>Location</b>: Baseline East</p><p><b>Recorder</b>: SYSTEM, SYSTEM Cerner</p><p><b>Performer</b>:</p><ul><li>Cerner Test, Physician - Women&apos;s Health Cerner</li></ul></div>' # rubocop:disable Layout/LineLength
```

Comment:
Can you wrap this text into multiple lines? So that you can remove rubocop disable.

Something like below:
'<div xmlns="http://www.w3.org/1999/xhtml"><p><b>Procedure</b></p>'\
'<p><b>Patient</b>: SMART, NANCY</p>'\

Reply:
comment addressed